tests/plans/test_topup_plan.py

```diff
@@ -15,6 +15,7 @@
 
 @pytest.fixture
 def synchrotron() -> Synchrotron:
+    RunEngine()
```

Comment:
Should we move hyperion's RE fixture to dodal? The

```
@pytest.fixture
def RE():
    RE = RunEngine({}, call_returns_result=True)
    RE.subscribe(
        VerbosePlanExecutionLoggingCallback()
    )  # log all events at INFO for easier debugging
    yield RE
    try:
        RE.halt()
    except Exception as e:
        print(f"Got exception while halting RunEngine {e}")
    finally:
        stopped_event = threading.Event()

        def stop_event_loop():
            RE.loop.stop()  # noqa: F821
            stopped_event.set()

        RE.loop.call_soon_threadsafe(stop_event_loop)
        stopped_event.wait(10)
    del RE
```

Reply:
Yes, probably, made https://github.com/DiamondLightSource/dodal/issues/449